src/components/state-details.js

```diff
@@ -27,21 +30,6 @@ import PowerPlantMap from "../images/dirty-power-plants.jpg"
 // New images
 import DirtyPowerPlantImg from "../images/dirty-power-plant.png"
 
-const slugToTitle = placeName => {
```

Comment:
Exctracted to new `helper-functions.js` since we need this on the power plant page too.